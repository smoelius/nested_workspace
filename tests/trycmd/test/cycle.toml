args = ["test", "--workspace"]

status = "failed"

[bin]
name = "cargo"

[fs]
cwd = "../../../fixtures/cycle"
