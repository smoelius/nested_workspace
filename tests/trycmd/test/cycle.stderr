...
error: failed to run custom build command for `cycle v0.1.0 ([CWD])`

Caused by:
  process didn't exit successfully: `[CWD]/target/debug/build/cycle-[..]/build-script-build` (exit [..]: 101)
  --- stdout
  cargo::rerun-if-changed=[CWD]/target/debug/build/cycle-[..]/out/[PUT].timestamp
...

  --- stderr
  <<< [CWD]/dependent
...
     Compiling cycle v0.1.0 ([CWD])
...
  [cycle 0.1.0] 
  [cycle 0.1.0] thread 'main' panicked at [..]/[PUT]/src/lib.rs:[..]:[..]:
  [cycle 0.1.0] called `Result::unwrap()` on an `Err` value: cycle detected: cannot run on nested workspaces
  [cycle 0.1.0] note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
  error: failed to run custom build command for `cycle v0.1.0 ([CWD])`

  Caused by:
    process didn't exit successfully: `[..] [CWD]/dependent/target/debug/build/cycle-[..]/build-script-build` (exit [..]: 101)
    --- stderr

    thread 'main' panicked at [..]/[PUT]/src/lib.rs:[..]:[..]:
    called `Result::unwrap()` on an `Err` value: cycle detected: cannot run on nested workspaces
    note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
  >>> [CWD]/dependent

  thread 'main' panicked at [..]/[PUT]/src/lib.rs:[..]:[..]:
  called `Result::unwrap()` on an `Err` value: command failed:[..] "cargo" "build" "-vv" "--offline" "--workspace"
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
